packages/wdio-concise-reporter/README.md

```diff
@@ -0,0 +1,18 @@
+WDIO Concise Reporter
+=====================
+
+> A WebdriverIO plugin to report in concise style.
+
+## Configuration
```

Comment:
How about adding an obligatory `Installation` step as well

Reply:
I'll add more detailed documentation it a bit later. Currently working on tests.